styles.css

```diff
@@ -31,20 +31,18 @@ main > section {
     margin: 0 auto;
 }
 
-section h3 {
-    margin-top: 100px;
+.imageContainer {
+    width: 750px;
 }
 
-.imageContainer {
-    width: 500px;
+section h3 {
+    margin-top: 100px;
 }
 
 img {
     width: 100%;
 }
 
-
-
 @media only screen and (max-width: 1500px) {
```

Comment:
Perhaps add another media query for different sizes - the title occupies two lines from 1200pixels.

Reply:
and again at 530pixels

Reply:
done